dfu/README.md

```diff
@@ -0,0 +1,59 @@
+## Description
+
+The DFU Update Module is to able to update peripheral devices connected to the device running Mender.
+
+Example use-cases:
+* Deploy firmware updates to peripheral devices using the USB Device Firmware Update (DFU) protocol
+
+For a more detailed tutorial on how to use this Update Module please visit [Mender Hub](https://hub.mender.io/t/device-firmware-update-dfu)
+
+### Specification
+
+|||
+| --- | --- |
+|Module name| dfu |
+|Supports rollback|no|
+|Requires restart|no|
+|Artifact generation script|yes|
+|Full system updater|no|
+|Source code|[Update Module](https://github.com/mendersoftware/mender-update-modules/tree/master/dfu/module/dfu), [Artifact Generator](https://github.com/mendersoftware/mender-update-modules/blob/master/dfu/module-artifact-gen/dfu-artifact-gen)|
+
+### Install the Update Module
+
+Download the latest version of this Update Module by running:
+
+```
+mkdir -p /usr/share/mender/modules/v3 && wget -P /usr/share/mender/modules/v3 https://raw.githubusercontent.com/mendersoftware/mender-update-modules/master/dfu/module/dfu && chmod +x /usr/share/mender/modules/v3/dfu
+```
+
+### Create artifact
+
+To download `dfu-artifact-gen`, run the following:
+
+```
+wget https://raw.githubusercontent.com/mendersoftware/mender-update-modules/master/dfu/module-artifact-gen/dfu-artifact-gen
+```
+
+Make it executable by running,
+
+```
+chmod +x dfu-artifact-gen
+```
```

Comment:
These could be a single command, same wayas we do for the Update Module itself, `wget ... && chmod +x ...`